### PR TITLE
Add ArgoAI Logo

### PR DESCRIPTION
**Pull request by rlee80**
#### Changes proposed in this pull request:

Addresses the task to "Add ArgoAI logo as a partner organization on the EvalAI landing page"

### CODE REVIEW

**Comment by Ram81** [comment]
@rlee7 can you share the screenshot of the homepage. In this one alignment looks messed up.

**Comment by rlee80** [comment]
> @rlee7 can you share the screenshot of the homepage. In this one alignment looks messed up.

image

https://pr-229-evalai.surge.sh/

**Comment by pushkalkatara** [comment]
Looks good to me. Nice work @rlee7

File: src/app/components/home/partners/partners.component.html [hunk 1]
```diff
@@ -32,6 +32,9 @@
 			<td>
 				<img class ="partner_img" src = "assets/images/partners/virginiatech.png"/>
 			</td>
+			<td>
+				<img class ="partner_img" src = "assets/images/partners/argoai.png"/>
+			</td>
 		</tr>
 	</table></div>
 </div>
```
Comment (RishabhJain2018) [inline src/app/components/home/partners/partners.component.html]: Please add a new line here.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can you please add an image with a transparent background.

File: src/app/components/home/partners/partners.component.html (hunk 1 above)
Comment (rlee80) [reply src/app/components/home/partners/partners.component.html]: > Please add a new line here.

Hey, I'm not sure what you mean by a new line, can you elaborate a little more? Thanks!

**Comment by rlee80** [comment]
> Can you please add an image with a transparent background.

Fixed!

File: src/app/components/home/partners/partners.component.html (hunk 1 above)
Comment (Ram81) [reply src/app/components/home/partners/partners.component.html]: @rlee7 can you add a new line after last `</div>`

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Comment by Ram81** [comment]
@rlee7 can you post the new screenshot?

**Comment by rlee80** [comment]
> @rlee7 can you post the new screenshot?

image

**Review by Ram81 — APPROVED** [review]
@rlee7 nice work. LGTM 👍

**Pull request closed without merging**